akka-docs/rst/scala/typed-actors.rst

```diff
@@ -1,6 +1,13 @@
+.. _typed-actors-scala:
+
 Typed Actors
 ====================
 
+.. note::
+
+  This module is deprecated as it will be superseded by the :ref:`typed-scala`
+  project which is currently being developed in open preview mode.
```

Comment:
+1
then there should be some deprecated annotations in akka.actor.TypedActor and a few words in the migration guide